source/manual/intro-to-docker.html.md

```diff
@@ -353,19 +353,18 @@ $mac docker-compose run content-publisher-demo bundle exec rake
 
 If we continue this iterative process add keep tidying up, adding more services, and consolidating the commands we have to run, like `bundle install` and `rake db:setup`, we end up [govuk-docker][govuk-docker] :-).
 
-## Summary
+## What now?
 
-In this tutorial you have been introduced to the concept of:
+In this tutorial we used Docker to create an environment for running tests against the Content Publisher repo. We started with a simple `docker run` command and added various flags to give us:
 
-* the `docker run` command
-* images (like a class)
-* containers (like instances of a class)
-* building images
-* creating and adding volumes
-* installing extra libraries with a Dockerfile
-* creating a Network
+* **volumes** the repo and gems
+* a **network** to communicate with Postgres
+* **environment variables** for Postgres
+* a **working directory** to reduce typing
 
-These concepts are useful to know when you start to use [govuk-docker], which you should use from this point on with GOV.UK.
+We also explored how to write and build an **image** that has the essential libraries and presets, using a **Dockerfile**. The end result is a single, relatively short command that we can use for day-to-day tasks, such as `bundle exec rake`.
```

Comment:
Perhaps encapsulate the above changes in a separate commit (and bump the last reviewed date on this file).

Then in the "Even More" commit, you can link to part 2.

Reply:
I've bumped the review dates. I think it makes sense to keep everything in one commit: I can't see a clear motivation to split it up, and keeping everything in one commit has the advantage of making it easy to revert, although I hope we won't need to do that!